src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -2428,8 +2664,78 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		return NULL;
```

Comment:
I missed that `TARGET_ARM64` is also only accelerating for `I1/U1` right now.

I'll get a commit up in the morning that adds the necessary support here as well. The logic we'd be copying over is https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/gentree.cpp#L25035-L25065, which isn't very complex overall.

Reply:
This should be handled now as well